### DISCUSSION TITLE
undo_log表rollback_info为什么要设置成longblob而不是longtext?

### DISCUSSION THREAD

**Discussion by lichenhui1234** [apache/incubator-seata#5496] [Q&A]
undo_log存储类型是longblob,查看源码发现是json序列化对象后再获取byte数组存入数据库,为什么不是拿序列化后的zfc直接存入数据库,存为longtext类型呢,这二者有什么区别吗

**Comment by slievrly** [comment]
longblob的设计主要考虑能存储原始字段为二进制的数据类型。
